Makefile.am

```diff
@@ -154,6 +154,8 @@ cpplint_FILES += $(noinst_HEADERS)
 clean-local:
 	-rm -f $(BUILT_SOURCES) $(CLEANFILES)
 
+check: cpplint
+
```

Comment:
The only difference I could find is that if you use `check-local` and `-j` it will run the cpplint check in parallel with other checks, while if you use `check` it will not (and will bail out if cpplint fails, before running any tests).